src/components/Tables/TableFilterAndExtraColumnsBar.tsx

```diff
@@ -117,14 +143,16 @@ const TableFilterAndExtraColumnsBar = <Data,>(props: Props<Data>): React.ReactEl
                     <FiltersAndIconContainer>
                         {hasAdditionalFilters && <FilterAltOutlined />}
                         <FilterContainer>
-                            {props.additionalFilters.length > 0 && (
-                                <>
-                                    {filtersToComponents(
-                                        props.additionalFilters,
-                                        props.setAdditionalFilters
-                                    )}
-                                </>
-                            )}
+                            {props.additionalFilters &&
+                                props.additionalFilters?.length !== 0 &&
```

Comment:
```suggestion
                            { props.additionalFilters?.length > 0 &&
```

Reply:
It doesn't recognise that additionalFilters is not undefined if I do this

Reply:
ah fair enough - happy to keep it as is!